rhizome/host/lib/vm_setup.rb

```diff
@@ -462,15 +462,15 @@ def generate_swap_config(swap_size_bytes)
     SWAP_CONFIG
   end
 
-  def write_user_data(unix_user, public_key, swap_size_bytes)
+  def write_user_data(unix_user, public_keys, swap_size_bytes)
     vp.write_user_data(<<EOS)
 #cloud-config
 users:
   - name: #{yq(unix_user)}
     sudo: ALL=(ALL) NOPASSWD:ALL
     shell: /bin/bash
     ssh_authorized_keys:
-      - #{yq(public_key)}
+#{public_keys.map { "      - #{yq(_1)}" }.join("\n")}
```

Comment:
Oh man, that we're still templating YAML (a prototype carryover) is such a bummer, but not for this patch to fix.